.github/workflows/pull-request-workflow.yml

```diff
@@ -62,3 +62,33 @@ jobs:
         run: yarn install --immutable
       - name: Build
         run: yarn build
+
+  tests:
+    needs:
+      - build
+    runs-on: ubuntu-latest
+    container: node:16-alpine3.15
+    services:
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_PASSWORD: example
+          POSTGRES_DB: test_integration
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+
+    steps:
+      - uses: actions/checkout@v3
+
```

Comment:
Please, when you are working in project/library/whatever with the code, always try to adjust your code to style guide in the project. The perfect code that you write is code that looks the same and couldn't be distinguish with code written previously. Given the current situation, look at the this file, do you see the place where we use space between lines?

Reply:
Ok, I'll remove space between lines